app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
Add type hints to the function to make it easier to read for other developers. You can use List from typing library the same as in `app/cafe.py` with dict. Using typing library allow you to be more specific with the types and avoid unwanted errors when using the function.

```suggestion
from typing import Dict, List

def go_to_cafe(friends: List[Dict[str, Any]], cafe: Cafe) -> str:
```